src/main/java/com/gmail/uramen66/ordermovies/service/TimesLotService.java

```diff
@@ -2,37 +2,52 @@
 
 import com.gmail.uramen66.ordermovies.dto.timesLot.TimesLotDTO;
 import com.gmail.uramen66.ordermovies.dto.timesLot.TimesLotMapper;
+import com.gmail.uramen66.ordermovies.exception.ResourceNotFoundException;
 import com.gmail.uramen66.ordermovies.model.TimesLot;
 import com.gmail.uramen66.ordermovies.repositories.TimesLotRepository;
 import lombok.AllArgsConstructor;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
 import org.springframework.stereotype.Service;
 
 @Service
 @AllArgsConstructor
 public class TimesLotService {
-    private TimesLotRepository timesLotRepository;
-    private TimesLotMapper timesLotMapper;
+    private final TimesLotRepository timesLotRepository;
+    private final TimesLotMapper timesLotMapper;
 
     public TimesLotDTO createTimesLot (TimesLotDTO timesLotDTO){
         TimesLot timesLot = TimesLot.builder()
 //                .startTime(timesLotDTO.getStartTime())
                 .build();
 
         TimesLot saveTimesLot = timesLotRepository.saveAndFlush(timesLot);
-        return timesLotMapper.timesLotToTamesLotDto(saveTimesLot);
+        return timesLotMapper.timesLotToTimesLotDto(saveTimesLot);
     }
 
-    public TimesLotDTO findById(Long id) throws Exception{
+    public TimesLotDTO findById(Long id) {
         return timesLotRepository
                 .findById(id)
-                .map(timesLotMapper::timesLotToTamesLotDto)
-                .orElseThrow(Exception::new);
+                .map(timesLotMapper::timesLotToTimesLotDto)
+                .orElseThrow(ResourceNotFoundException ::new);
     }
-    public void deleteTimesLot(Long id) throws Exception{
+    public void deleteTimesLot(Long id) {
         timesLotRepository.delete(
                 timesLotRepository.findById(id)
-                .orElseThrow(Exception::new)
+                .orElseThrow(ResourceNotFoundException ::new)
```

Comment:
The same as above for some description. It would be great to add description for all exception.

Reply:
For all class: movie, user and other? Or for all exception in class, vhere we use one's: delete, update, findById in all class?

Reply:
For all exception in class